packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -27,12 +27,12 @@ const defaultProps = {
 const cx = classNames.bind(styles);
 
 const NavLogo = ({
-  path,
+  url,
   altText,
   isCard,
   ...customProps
 }) => {
-  const image = <img className={cx('img')} src={path} alt={altText} />;
+  const image = <img className={cx('img')} src={url} alt={altText} />;
```

Comment:
I'm actually wondering if we shuld just have the navLogo be a react component.... Then all the link, card, and other logic can be handled outside of the nav framework.